.github/workflows/azure-static-web-apps-purple-desert-05060ea03.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, closed]
     branches:
+      - next.js
```

Comment:
why was this added?

Reply:
this was to run the test when i create a pr off this branch, because it was previously only set to main

Reply:
removed